src/baxter_pykdl/baxter_pykdl.py

```diff
@@ -143,30 +152,29 @@ def inverse_kinematics(self, position, orientation=None, seed=None):
         result_angles = PyKDL.JntArray(self._num_jnts)
 
         if self._ik_p_kdl.CartToJnt(seed_array, goal_pose, result_angles) >= 0:
-            result = np.array(result_angles)
+            result = np.array(list(result_angles))
             return result
         else:
-            print 'No IK Solution Found'
```

Comment:
When an optimization algorithm computes many IKs as fast as possible it floods the terminal and you have no way to get rid of that message cleanly with a Bash redirection nor ROS redirection in launch files.

The Python print should be at least been replaced by a rospy.loginfo so that the user can redirect the stream to void if he's uninterested. But I wouldn't even print anything here, in any case the user *must* check if the returned solution is None, he's then free to print whatever message he needs.